src/LanguageSelect.tsx

```diff
@@ -1,26 +1,22 @@
 import { Select } from "baseui/select";
 import { useState } from "react";
 import { useTranslation } from "react-i18next";
-import languages from "./translations/resources";
+import languages, { Language } from "./translations/resources";
 import { enqueueAnalyticsEvent } from "./utils/analytics";
 
+
 const LanguageSelect = () => {
-  const langCode = localStorage.getItem("i18nextLng") || "en";
-  const [language, setLanguage] = useState([
-    languages.find((lang) => lang.code === langCode),
-  ]);
+  const [language, setLanguage] = useState<Language | undefined>(
+    languages.find((lang) => lang.code === "en")
+  );
   const { i18n } = useTranslation();
 
   const changeLanguage = (selectedLanguage: any) => {
     let newLang = languages.find((lang) => lang.code === selectedLanguage.code);
-    setLanguage([newLang]);
+    setLanguage(newLang);
     i18n.changeLanguage(newLang?.code);
-
-    // Persist to local storage.
-    localStorage.setItem("i18nextLng", selectedLanguage.code);
```

Comment:
We should be able to leave this deleted as it is redundant since react-i18next already saves this to local storage.

Reply:
Sounds good!